### ISSUE TITLE
Clarification on implementation for convergence

### ISSUE THREAD

**Issue by ADH-LukeBollam** [sdoria/SimpleSelfAttention#4]
Hi Sebastian, thanks for the layer.

Can I clarify what you mean by: 
"the network converges if SimpleSelfAttention is placed right after a convolution layer that uses batch norm"

Are you saying the order of operations should always be

Conv => Batchnorm => Self Attention ?

And activation should come afterwards?

**Comment by carbocation** [comment]
Without commenting on what the order of operations *should always be*, the [implementation in this repo](https://github.com/sdoria/SimpleSelfAttention/blob/master/xresnet.py#L114) is consistent with:

Conv2D -> Self Attention -> Activation
